### PR TITLE
Introduce v3 batch event endpoint to Python SDK

### PR DESCRIPTION
**Pull request by mauerbac**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -9,4 +9,4 @@ MANIFEST
 dist
 
 # Output of running coverage locally
-.coverage
+.coverage
```
Comment (aliabbasrizvi) [inline .gitignore]: nit. Can you please undo this change

Comment (mauerbac) [inline .gitignore]: Done
